sympy/stats/crv_types.py

```diff
@@ -1353,8 +1360,8 @@ def GammaInverse(name, a, b):
     Parameters
     ==========
 
-    a : Real number, `a > 0` a shape
-    b : Real number, `b > 0` a scale
+    alpha : Real number, `\alpha > 0` a shape
```

Comment:
This should agree with the call parameter of GammaInverse and _argnames of GammaInverseDistribution.

Reply:
Ok, I'll change it